.circleci/create_circleci_config.py

```diff
@@ -189,6 +190,7 @@ def job_name(self):
         "sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev espeak-ng",
         "pip install --upgrade pip",
         "pip install .[sklearn,torch,testing,sentencepiece,torch-speech,vision,timm]",
+        "pip install natten",
```

Comment:
Ideally we can remove this

Reply:
Thank you, please refer to my comment below.